docs/installation.rst

```diff
@@ -47,33 +47,69 @@ Please add the channel conda-forge to your defaults::
 
 The conda-forge channel must be on top of the list before the anaconda default channel.
 
+install
++++++++
+
 You must install mss into a new environment to ensure the most recent
 versions for dependencies (On the Anaconda Prompt on Windows, you have to 
 leave out the 'source' here and below). ::
 
     $ conda create -n mssenv mamba
     $ conda activate mssenv
-    $ mamba install mss
+    $ (mssenv) mamba install mss
 
 
 You need to reactivate after the installation once the environment to setup all needed enironment
 variables. ::
 
     $ conda deactivate
     $ conda activate mssenv
-    $ mss
+    $ (mssenv) mss
 
 
+update
```

Comment:
I am unsure if we should not *only* recommend creating a new environment.

Reply:
at the moment we can update by mss this is solved too.